assets/scss/pages/_home.scss

```diff
@@ -1,3 +1,8 @@
 // -----------------------------------------------------------------------------
 // This file contains styles that are specific to the home page.
 // -----------------------------------------------------------------------------
+[class^="col"] {
```

Comment:
We need something more specific to tie this to the home page, such as a class on the body or html elements. Otherwise we can make it a more general style.

Reply:
Added a `.home` class